### ISSUE TITLE
[A11Y] "Skip access checks" checkbox not associated with label

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5951]
**Describe the bug**

The checkbox has a visible label but it is not semantically linked to it.

**Screenshots**

<img width="267" alt="image" src="https://github.com/indico/indico/assets/65413/2e57bf05-0899-4dc1-a353-daf533d47f7f">

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions
